build/jobs/run-export-tests.yml

```diff
@@ -171,7 +177,7 @@ jobs:
     displayName: 'Export E2E ${{ parameters.version }} SQL'
     inputs:
       command: test
-      arguments: '"**\*${{ parameters.version }}.Tests.E2E*.dll" --filter "FullyQualifiedName~SqlServer&Category=ExportLongRunning"'
```

Comment:
Should it be an "\" (inverted slash) or the "/" (regular slash)?
I think that it was doing the inverted slash previously.

Reply:
sorry, to clarify, did you mean for the removed code or new code in this comment?

Reply:
Not a big problem. I was just curious that the previous code was using inverted slashes and the new one is using regular slashes. I was wondering which one if the correct. 

But I also see that previous code in e2e-tests.yml is using regular slashes. 

I think everything is fine.

Reply:
I've been moving most things to `/` since its more compatible between windows/linux